app/views/episodes/media/_segment_count.html.erb

```diff
@@ -0,0 +1,11 @@
+<div class="col-12 mb-4 prx-field-group">
+  <div class="form-floating" data-controller="dynamic-form">
+    <%= form.number_field :ad_breaks, min: 0, max: Episode::MAX_SEGMENT_COUNT - 1, data: {action: "dynamic-form#change"} %>
```

Comment:
I think there is a code format issue on the spaces and curly braces?

Reply:
Err... where?  I'm not sure I follow.